pennylane/templates/layers.py

```diff
@@ -71,16 +72,14 @@ def StronglyEntanglingLayer(weights, wires, r=1, imprimitive=CNOT):
     Raises:
         ValueError: if less than 2 wires were specified
     """
-    if len(wires) < 2:
```

Comment:
Just curious, is this check/error not important?

Reply:
I changed the signature of StronglyEntanglingLayers to also work for 1 qubit...But I forgot to delete the ValueError in the docstring, so thanks!